tony-core/src/main/avro/ApplicationInited.avsc

```diff
@@ -5,6 +5,7 @@
   "fields": [
     {"name": "applicationId", "type": "string"},
     {"name": "numTasks", "type": "int"},
-    {"name": "host", "type": "string"}
+    {"name": "host", "type": "string"},
```

Comment:
we can refactor [host, container id] into a separate scheme like ContainerInfo as this is used in both AM and workers

Reply:
could be a follow up PR

Reply:
Yes will do in follow up PR